cve_bin_tool/cve_scanner.py

```diff
@@ -204,28 +204,36 @@ def get_cves(self, product_info: ProductInfo, triage_data: TriageData):
 
             if cves:
                 self.products_with_cve += 1
-                self.logger.info(f"Known CVEs in {product_info}")
+
+                # TODO: code below should be enabled in verbose
+                # self.logger.info(f"Known CVEs in {product_info}")
+
                 # error_mode.value will only be greater than 1 if quiet mode.
                 if self.error_mode.value > 1:
-                    title = f"[default][b]{len(cves)}[/b] CVE(s) in [b]{product_info.vendor}.{product_info.product}[/b] v[b]{product_info.version}[/b]"
-                    self.CONSOLE.log()
-                    self.CONSOLE.log(
-                        Panel(
-                            Columns(
-                                (
-                                    linkify_cve(
-                                        f"[{cve.severity.lower()}]{cve.cve_number}"
-                                    )
-                                    for cve in cves
-                                ),
-                                padding=(0, 2),
-                            ),
-                            padding=1,
-                            border_style="red",
-                            title=title,
-                        )
+                    # TODO: implement verbose mode with the code below
+
+                    # title = f"[default][b]{len(cves)}[/b] CVE(s) in [b]{product_info.vendor}.{product_info.product}[/b] v[b]{product_info.version}[/b]"
+                    # self.CONSOLE.log()
+                    # self.CONSOLE.log(
+                    #     Panel(
+                    #         Columns(
+                    #             (
+                    #                 linkify_cve(
+                    #                     f"[{cve.severity.lower()}]{cve.cve_number}"
+                    #                 )
+                    #                 for cve in cves
+                    #             ),
+                    #             padding=(0, 2),
+                    #         ),
+                    #         padding=1,
+                    #         border_style="red",
+                    #         title=title,
+                    #     )
+                    # )
+                    # self.CONSOLE.log()
```

Comment:
```suggestion
```

We can always get the old code out of git, so this can be removed rather than commented out.  Since all of this information is reported at the end I honestly don't think we'll be bringing it back unless a user specifically requests it.